web/index.html

```diff
@@ -1,28 +1,236 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+    <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
```suggestion
    <meta property="og:title" content="PyLadiesCon - Official Website of PyLadies Global Conference" />
    <meta property="og:type" content="website" />
    <meta property="og:image" content="http://localhost:63342/global-conference/web/img/pyladies_logo.png" />
    <meta property="og:url" content="https://conference.pyladies.com" />
```

Reply:
Here's an example of opengraph header tags. Maybe should be in a template? The url and title of each page should be different though.

Reply:
That's right. Adding this atm and the others will need to be modified.